.pre-commit-config.yaml

```diff
@@ -0,0 +1,24 @@
+repos:
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
+    rev: v0.5.5  
+    hooks:
+      - id: ruff
+        args: [--fix]
+        files: ^space2stats_api/
+
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v1.11.1   
+    hooks:
+      - id: mypy
+        args: [--ignore-missing-imports]
+        files: ^space2stats_api/
+        exclude: ^space2stats/env/
+
+  - repo: local
+    hooks:
+      - id: pytest
+        name: Run pytest
+        entry: bash -c 'PYTHONPATH=space2stats_api python -m pytest space2stats_api'
+        language: system
+        pass_filenames: false
+        always_run: true
```

Comment:
I would personally not run the test in the pre-commit step!

We usually use pre-commit only for formatting because it's a pretty fast step

Reply:
I can remove it! With the mocks, the unittests are fast.